recipes/glibmm/all/conanfile.py

```diff
@@ -47,7 +47,7 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("glib/2.78.3", transitive_headers=True)
+        self.requires("glib/2.81.0", transitive_headers=True)
```

Comment:
```suggestion
        if Version(self.version) >= "2.82.0":
            self.requires("glib/2.81.0", transitive_headers=True)
        else:
            self.requires("glib/2.78.3", transitive_headers=True)
```
Keep v2.78.3 for older versions to avoid conflicts with other recipes on CCI. That version is very widespread as a requirement.